hoomd/md/constrain.py

```diff
@@ -290,19 +290,20 @@ def __init__(self):
                 'constituent_types': [str],
                 'positions': [(float,) * 3],
                 'orientations': [(float,) * 4],
-                'charges': [float],
-                'diameters': [float]
             }),
                                      allow_none=True),
                               len_keys=1))
         self._add_typeparam(body)
         self.body.default = None
 
-    def create_bodies(self, state):
+    def create_bodies(self, state, charges=None):
         r"""Create rigid bodies from central particles in state.
 
         Args:
             state (hoomd.State): The state in which to create rigid bodies.
+            charges (dict[str, list[float]], optional): The charges for each of the
+                constituent particles, defaults to ``None``. If ``None``, all
+                charges are zero. The keys should be the central particles.
```

Comment:
```suggestion
            charges (dict[str, list[float]], optional): The charges for each of
                the constituent particles, defaults to ``None``. If ``None``,
                all charges are zero. The keys should be the central particles.
```
fix: `hoomd/md/constrain.py:304:81: E501 line too long (83 > 80 characters)`